### ISSUE TITLE
it-All out of date

### ISSUE THREAD

**Issue by jotonedev** [tvheadend/dtv-scan-tables#147]
Some of the frequencies listed are no longer in use because of the introduction of 5G in europe (https://ec.europa.eu/commission/presscorner/detail/hr/MEMO_16_206).

**Comment by Lalarian** [comment]
Yes, it sucks that tvheadend doesn't update mux lists anymore and isn't able to find any services without.
I even provided update lists, but just once got some feedback, having to upload it somewhere else that I neither could make sure if it was the right address, not could register or even participate there.. that's not very open or resourceful.
